### PR TITLE
Map classes and methods relating to powder snow and freezing

### PR DESCRIPTION
**Pull request by haykam821**

### CODE REVIEW

File: mappings/net/minecraft/PowderSnowBucketItem.mapping [hunk 1]
```diff
@@ -0,0 +1,6 @@
+CLASS net/minecraft/class_5634 net/minecraft/PowderSnowBucketItem
```
Comment (YanisBft) [inline mappings/net/minecraft/PowderSnowBucketItem.mapping]: Needs to be moved to nm.item

File: mappings/net/minecraft/block/LeveledCauldronBlock.mapping [hunk 2]
```diff
@@ -0,0 +1,17 @@
+CLASS net/minecraft/class_5556 net/minecraft/block/LeveledCauldronBlock
+	FIELD field_27206 LEVEL Lnet/minecraft/class_2758;
+	FIELD field_27880 RAIN_PREDICATE Ljava/util/function/Predicate;
+	FIELD field_27881 SNOW_PREDICATE Ljava/util/function/Predicate;
+	FIELD field_27882 precipitationPredicate Ljava/util/function/Predicate;
+	METHOD <init> (Lnet/minecraft/class_4970$class_2251;Ljava/util/function/Predicate;Ljava/util/Map;)V
+		ARG 1 settings
+		ARG 2 precipitationPredicate
+		ARG 3 behaviorMap
+	METHOD method_31650 subtractWaterLevel (Lnet/minecraft/class_2680;Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;)V
```
Comment (YanisBft) [inline mappings/net/minecraft/block/LeveledCauldronBlock.mapping]: `subtractFluidLevel` or `subtractLevel`?

File: mappings/net/minecraft/item/Items.mapping [hunk 3]
```diff
@@ -237,6 +237,7 @@ CLASS net/minecraft/class_1802 net/minecraft/item/Items
 	FIELD field_27075 CUT_COPPER Lnet/minecraft/class_1792;
 	FIELD field_27076 LIGHTLY_WEATHERED_CUT_COPPER Lnet/minecraft/class_1792;
 	FIELD field_27077 SEMI_WEATHERED_CUT_COPPER Lnet/minecraft/class_1792;
+	FIELD field_27876 POWDER_SNOW_BUCKET Lnet/minecraft/class_1792;
```
Comment (YanisBft) [inline mappings/net/minecraft/item/Items.mapping]: It's automapped for me

File: mappings/net/minecraft/PowderSnowBucketItem.mapping (hunk 1 above)
Comment (haykam821) [reply mappings/net/minecraft/PowderSnowBucketItem.mapping]: Fixed in 9c5142df05762088a9a6608e81dc2df8c3518239

File: mappings/net/minecraft/block/LeveledCauldronBlock.mapping (hunk 2 above)
Comment (haykam821) [reply mappings/net/minecraft/block/LeveledCauldronBlock.mapping]: Fixed in 49d20674ee2ce67d420976d1efd48442dcd74db9

**Review by YanisBft — APPROVED** [review]
Approving but I don't see the point of mapping automapped items

File: mappings/net/minecraft/item/Items.mapping (hunk 3 above)
Comment (Juuxel) [reply mappings/net/minecraft/item/Items.mapping]: Yep. Aren't the mappings for `Item` instances only for blocks since they don't get automapped?

Comment (Juuxel) [inline mappings/net/minecraft/item/Items.mapping]:
```suggestion
```

**Review by Juuxel — CHANGES_REQUESTED** [review]

**Review by Shnupbups — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**